docs/readmes/nms/userguide.md

```diff
@@ -0,0 +1,159 @@
+    * Button to **add new subscribers.**
+* ![subscriber_overview2](assets/nms/userguide/subscriber_overview2.png)
+
+### Subscriber Detail
+* Each Subscriber Detail page comprises of Overview, Events and Config section.
+* Overview
+    * Information section which displays Name, IMSI of the subscriber
+    * Status KPIs which displays GatewayID,  UE latency(if available), connection status(TBD), eNodeB Serial(TBD)
+    * Chart which displays the upload and download data usage for the subscriber in stacked bar chart.  It comes along with a date time selector to filter the chart within a specific time window.
+    * Data usage KPIs to show the data used by the subscriber in last 1 hour, 1 day, 1 week and 1 month time range.
+    * Event table which displays the subscriber specific events in last 3 hours.
+
+* Events
+    * Event chart displays the count of subscriber specific events in a bar graph.
+    * Event table which displays the time, event type and event description. It comes along with a dropdown to provide detailed dump of the event description. Additionally it has a search button which can be used to filter specific event types.
+    * Event page comes with a datetime selector which can be used to filter event chart and event table within a selected time window.
+
+* Config contains
+    * Subscriber section to view and configure LTE service state, Data plan, Auth Key and Auth OPC and Active APN associated with the subscriber
+    * Traffic section to view and edit(TBD) the APN and policy information associated with the subscriber
+
+* ![subscriber_detail1](assets/nms/userguide/subscriber_detail1.png)
+* ![subscriber_detail2](assets/nms/userguide/subscriber_detail2.png)
+* ![subscriber_detail3](assets/nms/userguide/subscriber_detail3.png)
+
+## **Traffic dashboard**
+
+### Policy dashboard
+* Policy Overview Table
+    This table contains all the policies configured in the network. Columns include PolicyID, flows, priority,  number of subscribers, monitoring key, rating, tracking type and actions. The actions performed on the individual rows include editing the JSON configuration, deactivating and removing a particular policy
+* ![policy_dashboard](assets/nms/userguide/policy_dashboard.png)
+
+### APN dashboard
+* APN Overview Table
+    This table contains all the APNs configured in the network. Columns include ApnID,  Description, Qos Profile,  Added Date, and Actions. The actions performed on the individual rows include editing the JSON configuration, deactivating and removing an APN
+* ![apn_dashboard](assets/nms/userguide/apn_dashboard.png)
+
+## Metrics dashboard(Grafana integration for superusers)
+* Same as before
+
+## Alerts dashboard
+* Same as before
+
+# Troubleshooting
+
+1. How to find all subscribers connected to a particular gateway ?
+2. How to get list of recent events in a particular gateway
+3. How to find all logs associated with a particular subscriber and export them ?
```

Comment:
Is this going to be filled in later? I think we should leave the section out if there aren't any answers

Reply:
I will add these details in. Forgot to add this.